docs/proposals/2019-04-23 - Audio.md

```diff
@@ -0,0 +1,50 @@
+# Problem
+
+Currently, audio objects are not in the Content API.  We are now at a point where we would like to add audio objects into the Content API and are honing the audio object to include all of the necessary elements into the json.  Thus, I propose that embed_html, series, and audio_type be added to the json.
+
+
+# Proposal
+
+Add embed_html, series, and audio_type to audio object.
+
+
+## Embed_html
+
+HTML for the embedded audio player
+
+## Series
+
+Unique ID for podcast parent objects.  A series is a collection of related podcasts.
+
+## Audio_type
+
+An enumeration of types of audio which will include a "podcast" as one of the types
```

Comment:
What will the other types be? I feel like other types might be less well-defined, in which case a new field isn't necessarily justified (could use `subtype`).